en/docs/setup/migration-guide.md

```diff
@@ -654,16 +654,48 @@ Follow the steps below to perform the upgrade.
           EXTENTSIZE 4;
     ```
 
-1.  Do the following database updates:  
+1.  If you manually added any custom OSGI bundles to the
```

Comment:
Shall we add a link to "Migrating custom components" section from here. Because, custom osgi's build with older versions may not work with 5.9.0